tests/Feature/Api/SettingsTest.php

```diff
@@ -62,14 +62,15 @@ public function testUpdateExtendedPropertiesWithValidVariableName()
     /**
      * Test extended properties variable invalid name validation
      */
-    public function testUpdateExtendedPropertiesWithInvalidVariableName()
+    public function updateExtendedPropertiesWithInvalidVariableName()
```

Comment:
Can we re-enable this test?

Reply:
It looks like i'm able to add spaces and hyphens using postman (PUT http://pmdev42.nossl/api/1.0/settings/89). It looks like you have back end validation for this in ProcessMaker/Providers/ProcessMakerServiceProvider.php so it should catch it right?

Reply:
Nolan, at first I added the backend validations, but there are some config data that should not pass that validations and there is no way to know if the config setting we are updating are from extended properties or another one. After review it with @caleeli we decide only keep the frontend validations, and the backend validation method is for the future if we need it.

Reply:
@nolanpro This validation fails when the setting page have empty values that does not pass the validation like in LDAP settings, so when you try to set any other property the validation throws an 422. That was why we decided to temporally disable that validation. Please take in count that when you enable back that validation.
![image](https://user-images.githubusercontent.com/8028650/144068319-7c97c714-d95e-480b-8043-6a35021e4911.png)

Reply:
@agustinbusso could you add `$this->markTestSkipped()` to the test and change the name back? It makes it easier to keep track of disabled tests

Reply:
Of course Nolan!